src/main/resources/db/changesets/20211209_datasetrequireslogging.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: datasetSecurity
+      author: tlangs
+      changes:
+        - dropColumn:
```

Comment:
I think this would be easier to understand if you removed the previous change set instead of adding and then dropping this column

Reply:
I've already tested this on deployed environments, so removing the change is a hassle.

Reply:
Ahh that's frustrating 😞 If you're talking about your personal deployment, you could modify the liquibase change log table if you don't want to drop your database.